### ISSUE TITLE
Creating new Review

### ISSUE THREAD

**Issue by griffind2009** [ga-wdi-exercises/project3#310]
Hi, I am trying to create a new review.  It saving to the database and I can see it the console when I log this.review, but it's not displaying on the show page. 

![screen shot 2016-11-02 at 8 49 29 am](https://cloud.githubusercontent.com/assets/20979785/19929344/4e4f0384-a0d9-11e6-87fc-45f28257467c.png)

![screen shot 2016-11-02 at 8 49 53 am](https://cloud.githubusercontent.com/assets/20979785/19929361/5a79c680-a0d9-11e6-84d3-9a609cf3e3d4.png)

**Comment by jshawl** [comment]
what happens when you visit the show page? any errors in the console?

**Comment by griffind2009** [comment]
No errors in the console just the saved object

**Comment by griffind2009** [comment]
![screen shot 2016-11-02 at 8 55 52 am](https://cloud.githubusercontent.com/assets/20979785/19929534/37bff172-a0da-11e6-9020-825344be4c4e.png)

![screen shot 2016-11-02 at 8 57 24 am](https://cloud.githubusercontent.com/assets/20979785/19929584/66b4151c-a0da-11e6-9cc1-c4c0d44c96d0.png)

**Comment by jshawl** [comment]
so the console shows the right information, but the browser does not?

**Comment by griffind2009** [comment]
yes, i can also see the object in the databse when i do rails c in the terminal

**Comment by jshawl** [comment]
ok what does your html look like for the show page?

**Comment by griffind2009** [comment]
![screen shot 2016-11-02 at 9 00 03 am](https://cloud.githubusercontent.com/assets/20979785/19929661/c439f54e-a0da-11e6-88d3-efe9c1ecbe01.png)

**Comment by jshawl** [comment]
it looks like you don't have anything inside the ng-repeat what if you put something between the last two tags

**Comment by griffind2009** [comment]
When I place in between the divs in ng-repeat nothing happened, but when I placed it above the div I got the following.

![screen shot 2016-11-02 at 9 07 51 am](https://cloud.githubusercontent.com/assets/20979785/19929864/f4ece9e8-a0db-11e6-9183-6b3e3d7489a1.png)

![screen shot 2016-11-02 at 9 07 22 am](https://cloud.githubusercontent.com/assets/20979785/19929912/2e5f4c20-a0dc-11e6-8168-fd7b7bd51d20.png)

![screen shot 2016-11-02 at 9 07 36 am](https://cloud.githubusercontent.com/assets/20979785/19929874/00e03836-a0dc-11e6-94d8-0fcc75544468.png)

**Comment by jshawl** [comment]
``` html
<div ng-repeat>
  put something here
</div>
```

**Comment by nolds9** [comment]
Which view are you on currently? What happens when you do `{{review.name}}` inside the `ng-repeat`?

**Comment by griffind2009** [comment]
I am on the show page. When I put something in ng-repeat I get an error that states there is an error in ng-repeats syntax.

**Comment by jshawl** [comment]
what is the error message can you share code and message?

**Comment by griffind2009** [comment]
![screen shot 2016-11-02 at 9 13 03 am](https://cloud.githubusercontent.com/assets/20979785/19930122/58f3d798-a0dd-11e6-82b0-eb4d227a4407.png)

![screen shot 2016-11-02 at 9 19 24 am](https://cloud.githubusercontent.com/assets/20979785/19930155/7bfb7cdc-a0dd-11e6-85fb-f2b841bc684e.png)

**Comment by griffind2009** [comment]
When i put 'review in vm.reviews in the div the error goes away

**Comment by nolds9** [comment]
yes, you need the full definition for `ng-repeat`

**Comment by jshawl** [comment]
``` html
<div data-ng-repeat="review in vm.reviews">
  {{review.name}}
</div>
```

**Comment by nolds9** [comment]
closing since we resolved in #312
